app/src/lib/git/log.ts

```diff
@@ -70,20 +70,24 @@ function mapStatus(
     return { kind: AppFileStatusKind.Deleted, submoduleStatus }
   } // deleted
   if (status === 'R' && oldPath != null) {
-    return { kind: AppFileStatusKind.Renamed, oldPath, submoduleStatus }
+    const renameIncludesModifications = false;
+    return { kind: AppFileStatusKind.Renamed, oldPath, submoduleStatus, renameIncludesModifications }
   } // renamed
   if (status === 'C' && oldPath != null) {
-    return { kind: AppFileStatusKind.Copied, oldPath, submoduleStatus }
+    const renameIncludesModifications = false;
+    return { kind: AppFileStatusKind.Copied, oldPath, submoduleStatus, renameIncludesModifications }
   } // copied
 
   // git log -M --name-status will return a RXXX - where XXX is a percentage
   if (status.match(/R[0-9]+/) && oldPath != null) {
-    return { kind: AppFileStatusKind.Renamed, oldPath, submoduleStatus }
+    const renameIncludesModifications = (status !== "R100");
+    return { kind: AppFileStatusKind.Renamed, oldPath, submoduleStatus, renameIncludesModifications }
   }
 
   // git log -C --name-status will return a CXXX - where XXX is a percentage
   if (status.match(/C[0-9]+/) && oldPath != null) {
-    return { kind: AppFileStatusKind.Copied, oldPath, submoduleStatus }
+    const renameIncludesModifications = false;
+    return { kind: AppFileStatusKind.Copied, oldPath, submoduleStatus, renameIncludesModifications }
```

Comment:
Nit: I think this would be cleaner if we did the assigning on the object 

{
 ... 
 renameIncludesModifications: false
 }